apps/framework-cli/src/framework/typescript/templates.rs

```diff
@@ -370,7 +370,7 @@ interface Aggregation {
 export default {
   select: ` 
     SELECT 
-        count(distinct userId) as dailyActiveUsers,
+        uniqState(userId) as dailyActiveUsers,
```

Comment:
`count distinct` operates on the entire data set at once. We probably saw it "work" because we already had data, then created the queries, which created the materialized view that looked properly aggregated.

But it doesn't aggregate new data because `count distinct` just doesn't support the incremental updates that a materialized view needs. Clickhouse needs an efficient way to track what's been aggregated and they do that through aggregation functions like `uniqState` & `uniqMerge`.